### ISSUE TITLE
Keystone of The Oblivion says nowhere that right clicking it over an item in your inventory deletes it.

### ISSUE THREAD

**Issue by UnstableStrafe** [Extegral/Enigmatic-Legacy#478]
The Keystone of The Oblivion at no point in the description or the wiki says that when you right click it over another item in your inventory, that item will be deleted. Also, it shouldn't be able to delete items with a lot of data in them like Backpacks or Bundles.

I had found the item in a chest and did the typical action of placing it into a Sophisticated backpack by hovering the item over my backpack in my inventory and right clicking. This deleted my backpack. The item does not say it does that. The wiki does not say it does that. I imagine this can easily happen if you tried to swap items in your inventory with it, only to lose something important like a dragon egg or a beacon. While I'm fairly sure there wasn't anything beyond a lot of random nether loot in that backpack, it feels extremely unfair to lose all those items from a feature that isn't even stated on the item itself.

There should at least be some kind of indication it does this or a way to retrieve the item if this happened.

**Comment by maxie13** [comment]
the exact same thing happened to me and unfortunately i had a bunch of important itens :( if this could be solved, it'd be awesome

**Comment by SodiumZH** [comment]
I also met this issue. Right click the backpack with an item typically means to put the item in, and this operation conflicted with the oblivion stone's deletion action. I lost 2 backpacks :(
Maybe setting the item disabled by default is better.

**Comment by SharonLougheed** [comment]
Agreed. Even if it's in disabled mode, it will still destroy everything you right click with it. Feels unfair to lose who knows what and how many hours because I didn't know this was set to a higher priority than backpacks. I get that this is a separate mod, but backpacks is a popular mod. Seems like accidentally deleting backpacks is a common experience. :( How many people don't have github who this happened to?

Intentional or not, this is so unintuitive and harmful that I think this should be labelled as _bug_ rather than _enhancement_. And if it _is_ intentional... not cool. But as-is, even if it's an otherwise useful item, I can't safely have that in my inventory, so I just left it in the loot chest.

Maybe make the default mode "Off" and rename "Disabled" to "On", so you can still right click to delete? Or _at very least_ make it lower priority than other right click functionality?

The relevant method is _overrideStackedOnOther_ in [src/main/java/com/aizistral/enigmaticlegacy/items/OblivionStone.java](https://github.com/Aizistral-Studios/Enigmatic-Legacy/blob/1.20.X/src/main/java/com/aizistral/enigmaticlegacy/items/OblivionStone.java).
```
	@Override
	public boolean overrideStackedOnOther(ItemStack stack, Slot slot, ClickAction action, Player player) {
		if (action == ClickAction.PRIMARY || !slot.mayPlace(stack) || !slot.mayPickup(player) || !slot.hasItem())
			return false;

		slot.set(ItemStack.EMPTY);

		if (player.level().isClientSide) {
			player.level().playSound(player, player.blockPosition(), SoundEvents.CHORUS_FRUIT_TELEPORT, SoundSource.PLAYERS, 0.25F, 1.2F + (float) Math.random() * 0.4F);
		}

		return true;
	}
```
So maybe there needs to be a mode check in that first if statement? But looking at _consumeStuff_ , mode 0 isn't _really_ disabled, so maybe could add another mode or something.

**Comment by PsiNAK** [comment]
Same thing just happened to me. Items like this should not exist. They cause so many players so much loss, and for what? How many players actually use it? And what is unique about it? Other items allow for much safer, more sensible voiding of items.

It is irresponsible to make an item like this without serious safeguards to protect against undesired loss.
